docs/source/markdown/podman-pod-logs.1.md

```diff
@@ -0,0 +1,92 @@
+% podman-pod-logs(1)
+
+## NAME
+podman\-pod\-logs - Displays logs for pod with one or more containers
+
+## SYNOPSIS
+**podman pod logs** [*options*] *pod*
+
+## DESCRIPTION
+The podman pod logs command batch-retrieves whatever logs are present with all the containers of a pod. Pod logs can be filtered by container name or id using flag **-c** or **--containername** if needed.
+
+Note: Long running command of `podman pod log` with a `-t` needs to be reinvoked if new container is added to the pod dynamically otherwise logs of newly added containers would not be visible in log stream.
```

Comment:
-t is for timeout, I think you mean --follow?

Note:  Running `podman pod log --follow ...` will not pick up newly added containers to the pod. Restart the command to see additional containers.